config/lib/systemd/system/plymouth-show-run.service

```diff
@@ -1,6 +1,5 @@
 [Unit]
 Description=Display state on splash
-After=plymouth-start.service
```

Comment:
I wonder if it is really okay to remove this "After=" relation - in the cases where plymouth is not started from the initrd we still need the ordering that plymouth-start.service is started first and then the services that use it are run. Or am I missing something here? AIUI having the After even if the service is not started is fine as it's just a weak dependency

Reply:
The ordering should be guaranteed as `ConditionPathExists=/run/plymouth/pid` applies in both cases. But you are right that adding `After=` should be fine too. I've put that back.